server/src/main/kotlin/com/cuhacking/watershed/util/JwtManager.kt

```diff
@@ -0,0 +1,32 @@
+package com.cuhacking.watershed.util
+
+import com.auth0.jwt.JWT
+import com.auth0.jwt.algorithms.Algorithm
+import com.cuhacking.watershed.config.Config
+import com.cuhacking.watershed.db.Database
+import io.ktor.auth.jwt.*
+import java.util.*
+import javax.inject.Inject
+
+class JwtManager @Inject constructor(private val config: Config, private val database: Database){
+
+    val algorithm: Algorithm = Algorithm.HMAC512(config.auth.secret)
+    private val TOKEN_EXPIRY = 60*60*1000 // 1 hour
+
+    fun createJWT(uuid: String) = JWT.create()
+        .withSubject("Authentication")
+        .withIssuer(config.auth.issuer)
+        .withClaim("uuid", uuid)
+        .withExpiresAt(getExpiration())
+        .sign(algorithm)
+
+    private fun getExpiration() = Date(System.currentTimeMillis() + TOKEN_EXPIRY)
```

Comment:
```suggestion
    private fun getExpiration() = Date(System.currentTimeMillis() + TOKEN_EXPIRY.toInt(DurationUnit.Milliseconds)
```
Could also add an extension function `Duration.toMilliseconds()` if that's too much to write lol